### PR TITLE
Migrate NpmAuthenticateV0 to Node 10

### PR DESCRIPTION
**Pull request by ghost**
**Task name**: NpmAuthenticateV0

**Description**: this pull request provides updating the NpmAuthenticateV0 task and its migration to Node 10

**Documentation changes required:** (Y/N) N

**Added unit tests:** (Y/N) N

**Attached related issue:** (Y/N) 

**Checklist**:
- [x] Task version was bumped - please check [instruction](https://github.com/microsoft/azure-pipelines-tasks/tree/master/docs/taskversionbumping.md) how to do it
- [x] Checked that applied changes work as expected

This task has been built and uploaded (via the tfx-cli tool) into the Azure DevOps organization and tested end-to-end with scenario from canary builds: https://github.com/microsoft/azure-pipelines-canary/tree/master/src/packaging/npmAuthenticate

### CODE REVIEW

**Review by anatolybolshakov — APPROVED** [review]

**Comment by tatyana-kostromskaya** [comment]
@aasim , @phil-hodgson could you please take a look at this PR?

**Comment by tatyana-kostromskaya** [comment]
cc @geekzter for visability

**Comment by tatyana-kostromskaya** [comment]
@aasim , @phil-hodgson could you please take a look at this PR?

**Review by phil-hodgson — APPROVED** [review]
LGTM

File: Tasks/NpmAuthenticateV0/package.json [hunk 1]
```diff
@@ -17,7 +17,13 @@
   },
   "homepage": "https://github.com/Microsoft/azure-pipelines-tasks#readme",
   "dependencies": {
-    "azure-pipelines-task-lib": "2.8.0",
-    "packaging-common": "file:../../_build/Tasks/Common/packaging-common-1.0.1.tgz"
```
Comment (tintse-thxsky-MSFT) [inline Tasks/NpmAuthenticateV0/package.json:21]: Why is the packaging-common being replaced with azure-pipelines-task-packing-common?

**Review by tintse-thxsky-MSFT — APPROVED** [review]

**Comment by tintse-thxsky-MSFT** [comment]
closing it. Will reopen by myself to sign the CLA

**Comment by mpodriezov** [comment]
@tintse-thxsky-MSFT  Please put a link to a new PR here,

**Comment by KonstantinTyukalov** [comment]
New PR: https://github.com/microsoft/azure-pipelines-tasks/pull/16650

**Pull request closed without merging**